### ISSUE TITLE
passKeyring does not implement Keyring (missing GetMetadata method)

### ISSUE THREAD

**Issue by chrisgilmerproj** [99designs/keyring#47]
While attempting to update my go dependencies for aws-vault and keyring I ran across this issue. 

```
$ go get github.com/99designs/keyring
go: finding github.com/99designs/keyring latest
# github.com/99designs/keyring
../../go/pkg/mod/github.com/99designs/keyring@v0.0.0-20190531235905-2e3b4e59b02e/pass.go:34:3: cannot use pass (type *passKeyring) as type Keyring in return argument:
        *passKeyring does not implement Keyring (missing GetMetadata method)
```

My guess is that #29 from @lox forgot to add this here and now it's a problem when loading it with go modules.

**Comment by philpennock** [comment]
#29 was my code and the problem is that `pass.go` did not exist when I wrote the PR, so I did not add support for the method.  Between my writing the patch and the patch being landed, more backends were added to `keyring`.

**Comment by chrisgilmerproj** [comment]
@philpennock - Would you be able to fix this? Or should I take a stab at it? In the interim it just means I can't get newer versions of this code through my build pipeline.

**Comment by chrisgilmerproj** [comment]
Actually, it appears that https://github.com/99designs/keyring/pull/46 addresses the issue. I'll wait to see if that gets merged.

**Comment by chrisgilmerproj** [comment]
Appears that this was resolved when #46 merged. Thanks all for your great work.
